generators/app/index.js

```diff
@@ -123,7 +114,7 @@ function showPrompts() {
  * Handles Node.js option.
  */
 function handleNodeJs(yo) {
-    var nodeJs = new NodejsHelper(addNodemon, portNumber, imageName);
+    var nodeJs = new NodejsHelper(portNumber, imageName);
 
     if (!nodeJs.canShareVolume()) {
```

Comment:
This only causes an issue if you are using nodemon which will only happen in debug. Should this be a warning instead?